i2c/26-i2c.html

```diff
@@ -79,7 +79,7 @@
             busno: {value:"1"},
             address: {value:"",required:false},
             command: {value:"",required:false},
-            count: {value:"1",required:true}
+            count: {value:"",required:true}
```

Comment:
Why would we not still default this to 1 ? User can remove if they need to override with msg.bytes. This is the default way nodes should work. If specified here it has precedence. Plus must not break existing instances.

Reply:
yes you are right, this was my mistake, it's better to leave 1 as default